docs/references/t-ecdsa-how-it-works.mdx

```diff
@@ -1,73 +1,114 @@
 import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
 
-# Threshold ECDSA
+# Threshold signatures
 
 <MarkdownChipRow labels={["Reference"]} />
 
 ## Overview
-This page gives a high-level outline of chain-key ECDSA signatures on ICP. Some of the information in this section is not required to use the feature, but may be of interest to the technically inclined reader for obtaining background information on the technology. ICP implements the threshold ECDSA protocol by Groth and Shoup as described in their [Eurocrypt 2022 paper](https://eprint.iacr.org/2021/1330). Groth and Shoup have also published a comprehensive [design and analysis](https://eprint.iacr.org/2022/506) of this distributed signing protocol.
+This page gives a high-level outline of threshold signatures on ICP. Some of the information in this section is not required to use threshold signing features, but may be of interest to the technically inclined reader for obtaining background information on the technology. ICP supports **threshold Schnorr** signing and **threshold ECDSA** signing, described by Groth and Shoup in their [Eurocrypt 2022 paper](https://eprint.iacr.org/2021/1330). Groth and Shoup have also published a comprehensive [design and analysis](https://eprint.iacr.org/2022/506) of this distributed signing protocol.
```

Comment:
For threshold Schnorr we could refer to this [paper](https://eprint.iacr.org/2023/1175), even though the implemented protocol is only the basic scheme sketched in the introduction, and not the full-fledged optimized protocol (yet). For security, there is also this other [paper](https://eprint.iacr.org/2023/1019) in which Victor analyses the security of several variations of Schnorr protocol, including the one used on the IC. 

Note that 506 is only about ECDSA and not Schnorr.

Reply:
Nice thanks. I'll expand the section, add those refs and make the separation.